### DISCUSSION TITLE
ADD-On Status "Dot"

### DISCUSSION THREAD

**Discussion by dCon0ne** [home-assistant/frontend#10282] [Supervisor]
![Add-on](https://user-images.githubusercontent.com/22716511/137524724-527385d2-5be3-4458-8190-bb32ce0e0ce4.JPG)
You know, when you go to the Add-On Dashboard, and click on an Add-On. In the upper right, of the Add-On Info pop-up, there is the Running/Stopped indicator.  It would be nice if that indicator where also on the Dashboard Add-On Title Carda
See attached photo markup.     After looking at the marked up photo, maybe place the status Dot, under the Add-On's Icon

**Comment by spacegaier** [comment]
I really like that idea. I am just wondering if the red dot might "scream" too much "ERROR". So an alternative would be to only show the green dot for the ones that are running, but not the red ones for the inactive ones...

I think I also once saw the idea floating that we add a color line to the top or bottom of an add-on tile (similar to the line we show when an update is available for the add-on or on the store page to show which are installed). We could then color that line red and green, but then we might need to have an icon/highlight for the ones that require an update to make them stand out again.

I also just saw that a red line is already used in the store for incompatible add-ons.

@matthiasdebaat Any wisdom from your side :) ?

**Reply by dCon0ne** [reply]
I see your point about "RED".   But "dot", "line", or "background' color change, doesn't matter to me, just would like to see a "QUICK Status"

**Reply by matthiasdebaat** [reply]
I like the idea to reuse the same status on the dashboard as on de detail page. Currently we grayed out a stopped add-on on the dashboard. When there's an update it got an orange line on the top. This isn't accessible for people who are colorblind. I agree with @spacegaier, icons that aren't immediately clear can cause confusion. A basic UX thing is to add a label, so in this case `Running`, `Stopped` and `Update`.

For [ESPHome](https://github.com/esphome/dashboard/pull/125) we've chosen to only show the deviations. So there is only a label when a device is offline, new or discovered. 

We've been working on an update of the supervisor dashboard. Will take a look how to include this.
